.github/workflows/entrypoint.sh

```diff
@@ -26,42 +27,83 @@ catch {set chains $env(POCKET_CORE_CHAINS)}
 set config ""
 catch {set config $env(POCKET_CORE_CONFIG)}
 
+set core_key ""
+catch {set core_key $env(POCKET_CORE_KEY)}
+
+set core_passphrase ""
+catch {set core_passphrase $env(POCKET_CORE_PASSPHRASE)}
+
 # Create dynamic config files
 if {$genesis != ""} {
-    set genesis_file [open /home/app/.pocket/config/genesis.json w]
+    set genesis_file [open "{$datadir}/config/genesis.json" w]
     puts $genesis_file $genesis
     close $genesis_file
     send_user "GENESIS loaded from env\n"
 }
 if {$chains != ""} {
-    set chains_file [open /home/app/.pocket/config/chains.json w]
+    set chains_file [open "${datadir}/config/chains.json" w]
     puts $chains_file $chains
     close $chains_file
     send_user "CHAINS loaded from env\n"
 }
 if {$config != ""} {
-    set config_file [open /home/app/.pocket/config/config.json w]
+    set config_file [open "${datadir}/config/config.json" w]
     puts $config_file $config
     close $config_file
     send_user "CONFIG loaded from env\n"
 }
 
-# If key isn't passed in, start the node
-if { $env(POCKET_CORE_KEY) eq "" }  {
+if {[regexp -nocase "datadir=" $command] && ![regexp -nocase "datadir=${datadir}" $command]} {
+  send_user "WARNING: --datadir provided with a different path to the one defined on Dockerfile. This could lead to errors when use this entrypoint to run CLI commands.\n"
```

Comment:
```suggestion
  send_user "WARNING: --datadir provided with a different path than the one defined in the Dockerfile. This could lead to errors when use this entrypoint to run CLI commands.\n"
```